src/components/WidgetWrapper.tsx

```diff
@@ -0,0 +1,88 @@
+import { createContext, PropsWithChildren, useEffect, useMemo, useRef, useState } from 'react'
+import styled from 'styled-components/macro'
+
+const HORIZONTAL_PADDING = 8
+
+const StyledWidgetWrapper = styled.div<{ width?: number | string }>`
+  -moz-osx-font-smoothing: grayscale;
+  -webkit-font-smoothing: antialiased;
+  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+  background-color: ${({ theme }) => theme.container};
+  border-radius: ${({ theme }) => theme.borderRadius}em;
+  box-sizing: border-box;
+  color: ${({ theme }) => theme.primary};
+  display: flex;
+  flex-direction: column;
+  font-size: 16px;
+  font-smooth: always;
+  font-variant: none;
+  min-height: 360px;
+  min-width: 300px;
```

Comment:
Should there be a max-width?

Reply:
that's not in the spec, but i'll check with fred and follow up with this if it turns out we do want it